compose/asc-image.c

```diff
@@ -212,8 +213,17 @@ asc_optimize_png (const gchar *fname, GError **error)
 	if (!asc_globals_get_use_optipng ())
 		return TRUE;
 
+	optipng_path = asc_globals_get_optipng_binary ();
+	if (optipng_path == NULL) {
+		g_set_error (error,
+			     ASC_IMAGE_ERROR,
+			     ASC_IMAGE_ERROR_FAILED,
+			     "optipng not found on $PATH");
```

Comment:
Nitpick: "in PATH" is probably better...

Reply:
yeah, typo I guess, that's what it is in the other places where I used a similar string